modules/conflict_resolver/php/conflict_resolver.class.inc

```diff
@@ -342,7 +341,6 @@ class Conflict_Resolver extends \NDB_Menu_Filter_Form
         // Project list, if applicable
         if ($config->getSetting('useProjects')==='true') {
             $projectList      = \Utility::getProjectList();
-            $list_of_projects = array('' => 'All') + $projectList;
             foreach ($projectList as $key => $value) {
                 $list_of_projects[$value] = $value;
```

Comment:
$list_of_projects needs to be defined now

Reply:
@zaliqarosli thanks!